docs/understanding-data-responses.md

```diff
@@ -0,0 +1,33 @@
+# Understanding Data Responses
+
+## Look up
+If it is ever unclear what an id number represents, you can use `lookup` to get all the information you'd need. `lookup` works for any of the [data series selection entities](./data-series-definition): `items`, `metrics`, `regions`, `frequencies`, `sources` and also `input_unit_id`. For example:
+```client.lookup('frequencies','7')```
+returns:
+```{'abbrev': None, 'id': 7, 'name': 'quarterly', 'periodLength': {'months': 3}}```
+
+```client.lookup('metrics','1480032')```
+returns:
+```py
+{'allowNegative': False,
+ 'allowedAggregations': '{sum,average}',
+ 'contains': [],
+ 'definition': 'The quantity of an item which has been consumed within a given country or region. Data generally refers to consumption as including any form of disappearance, such as waste, loss, and human consumption.',
+ 'id': 1480032,
+ 'name': 'Domestic Consumption (mass)',
+ 'rankingScore': 1}

```

Comment:
`rankingScore` isn't included in this output anymore

Reply:
removed